src/_includes/config/es-version-23.md

```diff
@@ -0,0 +1,9 @@
+{:.bs-callout-warning}
+{{site.data.var.ee}} 2.3.6 supports Elasticsearch 7.x.x. (default).
+Both Elasticsearch 2.x and 5.x have reached [end-of-life][] and are no longer supported. Support for Elasticsearch 6.8 has been deprecated in Magento 2.3.6, but you can still use it in Magento 2.3.5 and 2.3.6.
```

Comment:
I would rewrite the warning as follows:

As of version 2.3.6, Magento supports Elasticsearch 7.x, with a preference for 7.6.x. Support for Elasticsearch 6.x has been deprecated, but it can still be used. Both Elasticsearch 2.x and 5.x have reached [end-of-life][] and are no longer supported.

Reply:
Thanks @keharper! I like your rewrite.